Compiler warnings

I don't know if any of the following indicate actual problems, but I thought I'd report them.
There are a bunch of warnings about indentation that sound like possible mistakes one would be prone to when switching between C and Python.

```bash
ross@barley:/usr/local/jar/OpenABM-Covid19/src$ date; time make all
Wed 10 Jun 2020 02:17:58 PM PDT
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c utilities.c -o utilities.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c constant.c -o constant.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c demographics.c -o demographics.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c params.c -o params.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c model.c -o model.o
model.c: In function ‘set_up_healthcare_workers_and_hospitals’:
model.c:708:9: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
         if( !individual_eligible_to_become_healthcare_worker( indiv ) )
         ^~
model.c:711:3: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘if’
   indiv->worker_type = DOCTOR;
   ^~~~~
model.c:727:9: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
         if( !individual_eligible_to_become_healthcare_worker( indiv ) )
         ^~
model.c:730:3: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘if’
   indiv->worker_type = NURSE;
   ^~~~~
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c individual.c -o individual.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c main.c -o main.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c input.c -o input.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c network.c -o network.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c disease.c -o disease.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c interventions.c -o interventions.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c hospital.c -o hospital.o
hospital.c: In function ‘release_patient_from_hospital’:
hospital.c:548:5: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
     if( indiv->ward_type != NO_WARD )
     ^~
hospital.c:551:2: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘if’
  indiv->ward_type = NO_WARD;
  ^~~~~
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c doctor.c -o doctor.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c nurse.c -o nurse.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c ward.c -o ward.o
gcc -g -Wall -fmessage-length=0 -I/usr/local/include -O0 -c list.c -o list.o
gcc -L/usr/local/lib -o covid19ibm.exe utilities.o constant.o demographics.o params.o model.o individual.o main.o input.o network.o disease.o interventions.o hospital.o doctor.o nurse.o ward.o list.o -lgsl -lgslcblas -lm -O3

real	0m2.217s
user	0m0.723s
sys	0m0.116s
ross@barley:/usr/local/jar/OpenABM-Covid19/src$ date; make install
Wed 10 Jun 2020 02:19:16 PM PDT
gcc -L/usr/local/lib -o covid19ibm.exe utilities.o constant.o demographics.o params.o model.o individual.o main.o input.o network.o disease.o interventions.o hospital.o doctor.o nurse.o ward.o list.o -lgsl -lgslcblas -lm -O3
swig -python covid19.i
constant.h:179: Warning 454: Setting a pointer/reference variable may leak memory.
python3 -m pip install --upgrade .
## OK beyond here
```

Running on Debian 10=buster.  More about the environment:
gcc 8.3.0-6
python 3.7.3
swig 3.0.12-3

I am not operating in a python virtual environment, though in retrospect perhaps that would have been a good idea.

--- a/src/hospital.c
+++ b/src/hospital.c
@@ -545,8 +545,8 @@ void release_patient_from_hospital( individual *indiv, hospital *hospital )
 {
 	remove_if_in_waiting_list( indiv, hospital );
 
-    if( indiv->ward_type != NO_WARD )
-        remove_patient_from_ward( &(hospital->wards[indiv->ward_type][indiv->ward_idx]), indiv );
+	if( indiv->ward_type != NO_WARD )
+		remove_patient_from_ward( &(hospital->wards[indiv->ward_type][indiv->ward_idx]), indiv );
 
 	indiv->ward_type = NO_WARD;
 	indiv->ward_idx  = NO_WARD;

--- a/src/model.c
+++ b/src/model.c
@@ -705,12 +705,12 @@ void set_up_healthcare_workers_and_hospitals( model *model)
 		pdx = gsl_rng_uniform_int( rng, model->params->n_total );
 		indiv = &(model->population[pdx]);
 
-        if( !individual_eligible_to_become_healthcare_worker( indiv ) )
-                continue;
+		if( !individual_eligible_to_become_healthcare_worker( indiv ) )
+			continue;
 
 		indiv->worker_type = DOCTOR;
 		indiv->occupation_network = HOSPITAL_WORK_NETWORK;
-        add_healthcare_worker_to_hospital( &(model->hospitals[0]), indiv, DOCTOR );
+		add_healthcare_worker_to_hospital( &(model->hospitals[0]), indiv, DOCTOR );
 		idx++;
 	}
 
@@ -724,12 +724,12 @@ void set_up_healthcare_workers_and_hospitals( model *model)
 		pdx = gsl_rng_uniform_int( rng, model->params->n_total );
 		indiv = &(model->population[pdx]);
 
-        if( !individual_eligible_to_become_healthcare_worker( indiv ) )
-                continue;
+		if( !individual_eligible_to_become_healthcare_worker( indiv ) )
+			continue;
 
 		indiv->worker_type = NURSE;
 		indiv->occupation_network = HOSPITAL_WORK_NETWORK;
-        add_healthcare_worker_to_hospital( &(model->hospitals[0]), indiv, NURSE );
+		add_healthcare_worker_to_hospital( &(model->hospitals[0]), indiv, NURSE );
 		idx++;
 	}
 }

